components/dashboard/src/components/RepositoryFinder.tsx

```diff
@@ -327,27 +337,58 @@ export default function RepositoryFinder({
                 });
             }
 
-            if (searchString.length >= 3 && authProviders.data?.some((p) => p.type === AuthProviderType.AZURE_DEVOPS)) {
-                // ENT-780
+            if (
+                searchString.length > 0 &&
+                searchString.length < 3 &&
+                authProviders.data?.some((p) => p.type === AuthProviderType.GITLAB) &&
+                !onlyConfigurations
+            ) {
+                // add an element that tells the user that GitLab only does exact searches for short queries
                 result.push({
-                    id: "azure-devops",
+                    id: "gitlab",
                     element: (
                         <div className="text-sm text-pk-content-tertiary flex items-center">
                             <Exclamation2 className="w-4 h-4 mr-2" />
-                            <span>Azure DevOps doesn't support repository searching.</span>
+                            <span>Search is too short for searching on GitLab.</span>
```

Comment:
IMO that message would confuse me, as I potentially see results. Should be mention exact matches?

Reply:
@geropl I see 2 options here:
1) only show this if the number of results is 0
2) mention exact matches _somehow_ (I'm having a hard time making the message succinct and clear at the same time). WDYT about this?

```
<span>Search text is too short. GitLab will only show exact matches for short searches.</span>
```

Reply:
I think that message is great! What about the even more specific form:

`<span>Search text is < 3 characters. GitLab will only show exact matches for short searches.</span>`

Reply:
purrfect! :shipit:

Reply:
Pushed in [`89221af` (#20415)](https://github.com/gitpod-io/gitpod/pull/20415/commits/89221af6dda9ae5df0786a3b63f4acb538c5143b)